pkg/helper/client.go

```diff
@@ -5,17 +5,33 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"os"
 	"os/exec"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	yaml "sigs.k8s.io/yaml"
 
+	"github.com/gobwas/glob"
 	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
 )
 
 const nmstateCommand = "nmstatectl"
 
+var (
+	interfacesFilter     string
```

Comment:
We can drop this variable, it is not used anywhere except the init. Only Glob is used